app/src/main/res/layout-hdpi/control_main_list_item.xml

```diff
@@ -54,28 +54,37 @@
             <EditText
                 android:id="@+id/edittext_control_main"
                 android:layout_width="0dp"
-                android:layout_weight="2.5"
-                android:layout_height="match_parent"
-                android:gravity="center"
+                android:layout_height="45dp"
```

Comment:
Use dimensions from resource files other than hard-coding them. Here and every other file.

Reply:
@CloudyPadmal  These were already present and I have not changed them. I recently changed the hard-coded values to the ones from the resource file in the Text View.